cves/CVE-2007-3847.yml

```diff
@@ -77,9 +81,12 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: |
+    From looking at the commits that introduced and fixed the vulnerability,
+    there are no evidence that tests of any kind were involved. Both commits
+    only changed files in the impacted module, proxy.
```

Comment:
What is the function of the proxy module in Apache? I can take a guess at it, but people who aren't familiar with web may not understand what the module does.

Reply:
While I agree with you that some people might not know what is the function of proxy in apache, this section is concerned with unit and automated tests.